opendocdb-cts/internal/mongosh/fixture_test.go

```diff
@@ -136,11 +152,15 @@ func TestConvertFixtures(t *testing.T) { //nolint:revive // exceeds number of li
 		"Decimal": {
 			fixtures: data.Fixtures{
 				"c": []*wirebson.Document{
-					wirebson.MustDocument("_id", "decimal128", "v", wirebson.Decimal128{H: 42, L: 13}),
+					wirebson.MustDocument(
+						"_id", "decimal128",
+						"v", wirebson.Decimal128{H: 3475653012423180288, L: 4213},
+					),
 				},
 			},
 			expected: `
 			db.c.insertMany([{"_id": "decimal128", "v": Decimal128("42.13")}]);`,
+			skip: "https://github.com/OpenDocDB/cts/issues/34",
```

Comment:
https://github.com/OpenDocDB/cts/pull/31/files#r2052033918

Why do we have two PR, one resolving that problem and another reintroducing it?